tests/e2e/gc_block_resize_retain_policy.go

```diff
@@ -275,12 +285,13 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Tests with reclaimation po
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		// Create a new POD to use this PVC, and verify volume has been attached
+		// Create a new POD to use this PVC, and verify volume has been attached.
```

Comment:
POD -> Pod.

Same comment in other places in this file. "POD" is not an abbreviation so it should be "Pod". Btw, this is not your change. Since you are cleaning up this file, you may have to do a find all and replace all.

Reply:
I prefer use a different change, otherwise it will snowball into a never-finishing effort.

As the matter of fact, @SandeepPissay and others, would any of you take the challenge to fix those issues as you see fit. We need helps from all to clean up the code. The horsepower of code cleanup and new feature development seems not well balanced.

Reply:
Ok, let me fix this issue.

Reply:
Thanks, Sandeep.

Reply:
Addressing it here - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1120

Reply:
Thanks, Sandeep.

It would be better to merge this change first. Otherwise, there will be a merge conflict down the road.